rc/filetype/just.kak

```diff
@@ -39,6 +39,8 @@ define-command -hidden just-indent-on-new-line %{
 
 add-highlighter shared/justfile regions
 add-highlighter shared/justfile/content default-region group
+add-highlighter shared/justfile/recipe region '^[\w-]+(\s.+)?\s?:(?!=).*' '$' group
```

Comment:
```suggestion
add-highlighter shared/justfile/recipe region ^[\w-]+(\h[^\n]*|\h?):(?!=) $ group
```

Reply:
@krobelus I think this was your suggestion. It looks good to me, will test shortly.